app/actions/note.js

```diff
@@ -97,13 +97,26 @@ export const deleteNote = () => (dispatch, getState) => {
 
   const state = getState().noteReducer
   const noteIndex = state.activeNoteIndex
-  removeNote(state.notes[noteIndex].id,
-    state.notes[noteIndex].rev).then((result) => {
-    dispatch(deleteNoteFromList(result.id))
-    dispatch(setActiveNoteIndex(ACTIONS.NOT_SELECTED_NOTE))
-    dispatch(setNoteStatus(NOTE_STATUS.NOTE_DELETE_SUCCESS))
-  }).catch((err) => {
-    dispatch(setNoteStatus(NOTE_STATUS.NOTE_DELETE_FAIL))
+  try {
```

Comment:
If there's a better approach to this, I'd like to know... So I can improve it :)

Reply:
I think this is not needed as you already have two catches. @enzo-beltrami What do you think?

Reply:
We have some logic under the second catch so if something goes wrong there it'll throw an Exception that will get caught in this catch but i think we should run  `    dispatch(setNoteStatus(NOTE_STATUS.NOTE_DELETE_FAIL))
` to give the user some feedback.